examples/resources/illumio-cloudsecure_k8s_cluster/resource.tf

```
@@ -0,0 +1,19 @@
+resource "illumio-cloudsecure_k8s_cluster" "example" {
+  client_id      = "xxxxx"
+  client_secret  = "xxxxx"
+  illumio_region = "aws-us-west-2"
+
+  # Optional attributes
+  log_level = "debug"
+}
+
+output "example_client_id" {
+  value       = illumio-cloudsecure_k8s_cluster.example.client_id
+  description = "The clusters Oauth2 client_id"
+}
+
+output "example_client_secret" {
+  value       = illumio-cloudsecure_k8s_cluster.example.client_secret
+  description = "The clusters Oauth2 client_secret"
+  sensitive   = true
+}
```

1. Keep the descriptions consistent with the example for `k8s_cluster_onboarding_credential`.
1. A description ends with a period.
```suggestion
output "example_client_id" {
  value       = illumio-cloudsecure_k8s_cluster.example.client_id
  description = "The client_id to use to authenticate this k8s cluster."
}

output "example_client_secret" {
  value       = illumio-cloudsecure_k8s_cluster.example.client_secret
  description = "The client_secret to use to authenticate this k8s cluster."
  sensitive   = true
}
```